examples/X_NUCLEO_IKS02A1_Audio_Button/X_NUCLEO_IKS02A1_Audio_Button.ino

```diff
@@ -0,0 +1,233 @@
+#include "Arduino.h"
+#include "app_config.h"
+#include "PDM.h"
+#include "pdm2pcm.h"
+#include "WaveEncoder.h"
+#include "SD.h"
+#include <CMSIS_DSP.h>
+
+/* PDM */
+uint16_t sampleBuffer[(((AUDIO_IN_FREQ / 8) * MAX_AUDIO_IN_CHANNEL_NBR_PER_IF * N_MS_PER_INTERRUPT)/2)] = {0};
+int count = 1;
+String file_name = "";
+
+/* Button */
+bool start_stop = false; /* true means start, false means stop */
+volatile bool button = false;
+
+const int buttonPin = USER_BTN;
```

Comment:
To avoid failed build in case of `USER_BTN` not defined you should define one by default if not exists, I set 5 but it can be an other depending of the shield pins:
```suggestion
#ifdef USER_BTN
const int buttonPin = USER_BTN;
#else
const int buttonPin = 5;
#endif
```